web/src/reducers/budget.js

```diff
@@ -133,7 +133,7 @@ const buildBudget = bigState => {
 
   // Since all of our expenses are tied up in activities, we'll start
   // by looking at all of them and doing Magic Math™. (It's not magic.)
-  Object.values(bigState.activities.byKey).forEach(activity => {
+  bigState.apd.data.activities.forEach(activity => {
```

Comment:
For the budget stuff, we also want to start using the activities from the APD rather than `byKey`.